### DISCUSSION TITLE
一般内群要多长时间才能通过

### DISCUSSION THREAD

**Discussion by SqrtTan** [Meloong-Git/PCL#3580] [提问] [answered]
如题，我几个小时还没过

**Comment by allMagicNB** [comment, marked as answer]
管理一般会在晚上放人，一天内肯定能过
总之，先不要急

**Reply by SqrtTan** [reply]
现在7点15还没过（

**Reply by wuliaodexiaoluo** [reply]
> 我发了加群申请，但是没有通过，我该怎么办？
如果申请在发送 24 小时后 依然没有被同意或拒绝，可能是由于你的 QQ 被标记为可疑账号，因此申请会被傻逼腾讯过滤……那么先试试用 手机以及电脑 各自重新发送一次申请，如果一天后还是没有通过，那你可能得换个 QQ 号了……
或者，请检查填写的入群验证码是否有误：验证码一般是一段乱码，不要错填成识别码、解锁码或是更新密钥。也请注意不要打错字（例如把小写 L 打成了大写 i），最好直接复制粘贴。

详见 《PCL 常见问题 兼 脑溢血急救中心》

**Reply by SqrtTan** [reply]
现在过了
